fluence-app-service/src/service.rs

```diff
@@ -43,8 +43,8 @@ impl AppService {
     /// Create Service with given modules and service id.
     pub fn new<C, S>(config: C, service_id: S, envs: HashMap<Vec<u8>, Vec<u8>>) -> Result<Self>
     where
-        C: TryInto<AppServiceConfig>,
         S: Into<String>,
+        C: TryInto<AppServiceConfig>,
```

Comment:
is this reordeting intentional?

Reply:
Yes, it's intended to keep lines related to one generic parameter near each other.